tests/installation/partitioning_raid.pm

```diff
@@ -100,7 +100,12 @@ sub addraid {
 
 sub setraidlevel {
     my ($level) = @_;
-    my %entry = (0 => 0, 1 => 1, 5 => 5, 6 => 6, 10 => 'g');
+    my %entry = (
+        0  => 0,
+        1  => 1,
+        5  => 5,
+        6  => 6,
+        10 => (sle_version_at_least('15') ? 'o' : 'g'));
```

Comment:
yay - TW is sle_version_at_least(15) == true...  but it needs 'g' to be pressed (there is no stoarge-ng in TW, which likely impacts this)